sechub-webui-solution/docker/clone.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
```

Comment:
Shouldn't we have this inside the shared script folder ?
Looks like it could be useful in other solutions as well.